android/app/build.gradle

```diff
@@ -148,7 +148,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "org.pathcheck.covidsafepaths.bt"
+        applicationId "org.pathcheck.la.bt"
```

Comment:
Lets not include these build specific changes.

Reply:
Not sure how those got there 👺